x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/print_layout.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import path from 'path';
+import { KbnServer } from '../../../../../../../../src/server/index';
+import { Size } from '../../../../../types';
+import { CaptureConfig } from './index.d';
+import { Layout } from './layout';
+
+type EvalArgs = any[];
+
+interface EvaluateOptions {
+  // Fn is a function in string form to avoid tslint from auto formatting it into a version not
```

Comment:
`Fn` => `fn` since it's referencing a variable, probably shouldn't be capitalized.   Maybe even add back tics around it.